pom.xml

```diff
@@ -468,25 +390,7 @@
           <OSType>mac</OSType>
       </properties>
     </profile>
-    <profile>
```

Comment:
win32 is old! Frankly, I don't even know if it will work in win 32bits.

Reply:
Exactly. Having such obsolete profiles is confusing at best, frustrating if the community members try to use them.